addon/helpers/union.js

```diff
@@ -1,4 +1,7 @@
-import { union } from '@ember/object/computed';
-import createMultiArrayHelper from '../-private/create-multi-array-helper';
+import { helper } from '@ember/component/helper';
 
-export default createMultiArrayHelper(union);
+export function union([...arrays]) {
+  return [...new Set([].concat(...arrays))];
```

Comment:
https://stackoverflow.com/a/27664971/10879402

Reply:
I'm not sure what IE11 compat should be but without a polyfill, `new Set(iterable)` isn't supported: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set#Browser_compatibility

Reply:
I assumed that babel would allow us to use `Set`, I'm not positive though.

Reply:
You *may* need to double-spread it? `new Set(...[].concat(...arrays))`.

Reply:
I've tried both `Set` approaches in IE11. As @makepanic pointed out, they don't work without a polyfill. Instead, I'm now using the [`[].filter` approach](https://stackoverflow.com/a/14438954/10879402):

```js
export function union([...arrays]) {
  let items = [].concat(...arrays);

  return items.filter((value, index, array) => array.indexOf(value) === index);
}
```